### ISSUE TITLE
Render all section of book

### ISSUE THREAD

**Issue by thuynt46** [futurepress/epub.js#1354]
I really want to find a solution to render all sections of the book. Can you give me some way?

**Comment by thuynt46** [comment]
Anyone hereeeeeee

**Comment by johnfactotum** [comment]
Loop through the sections (using e.g. `book.spine.each()`) and call `section.render()` to get the HTML.

**Comment by thuynt46** [comment]
@johnfactotum uhm, my idea is rendering all the sections. After that base on location calculate the total page. Is it Is it possible or not? My goal is to calculate totalpage.

**Comment by johnfactotum** [comment]
Yeah, if you only want to calculate the number of pages, you just render each section and add up `location.start.displayed.total`.

**Comment by thuynt46** [comment]
@johnfactotum Thanks for your answers. I did this. It's working nicely in a book with the book separated into multiple sections. And each section has a page count of less than 50 pages. And it is very slow for books with content each section is about 500 pages large. Is there a way to optimize the calculated total number and render section time?

**Comment by johnfactotum** [comment]
AFAIK, not possible.
